gen_cargo_config.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/python
```

Comment:
On that note, a python version should probably be specified. I wasn't sure if ubuntu still defaulted to python so i tested it out by installing python 2 and 3 into a ubuntu:20.04 docker container. At least in the container, it doesn't default to any version, making the "python" command non-existant.

I personally also prefer using `#!/usr/bin/env python3` (if 3 is chosen) since a python installation from source would for example be installed to `/usr/local/bin/` instead. This makes `env` find the first working python3 binary according to the path and I never stumbled upon any installation that didn't have `env` installed/symlinked in that location.